### ISSUE TITLE
ERROR in ./node_modules/buffermaker/test/BufferMaker.js Module not found: Error: Can't resolve 'should'

### ISSUE THREAD

**Issue by harry524483** [cainus/BufferMaker#9]
When using webpack to build your library we get the the error: ERROR in ./node_modules/buffermaker/test/BufferMaker.js
Module not found: Error: Can't resolve 'should'

This is because 'should' is only a a dev dependency of your project and you have included you tests in the distributed package.

Could you remove your tests from the distributed package or add 'should' to you dependencies?

Thanks Harry

**Comment by slinkardbrandon** [comment]
I just ran into this issue as well.

**Comment by harry524483** [comment]
We got this issue while using "kafka-node" which uses "buffermaker" as a dependency. We resolved it by adding "should" library in our package.json.

**Comment by slinkardbrandon** [comment]
Yeah, I also got it by using `kafka-node`. I don't want to add should to the package.json as it _should_ be a dev dependency.

**Comment by DaTebe** [comment]
When can we expect the fix?

**Comment by slinkardbrandon** [comment]
I opened a PR to resolve this issue 3 months ago. 

See the referenced issue above (or this link): https://github.com/cainus/BufferMaker/issues/10

~but it’s been ignored~

It was just merged. In the above PR @cainus also mentioned that they'd be flagging this repo as unmaintained.
